ProjFS.Mac/PrjFSKext/PrjFSKext/KauthHandler.cpp

```diff
@@ -284,10 +296,56 @@ static int HandleVnodeOperation(
                 }
             }
         }
+        else if (ActionBitIsSet(action, KAUTH_VNODE_DELETE))
```

Comment:
One more question - do we need to trigger a recursive hydration of this directory? What's the current behavior if you move a virtual directory? What if you move a directory that is non-empty but has virtual items nested deep within it?

Reply:
I'll leave this investigation to the PR that adds support for the rename notification.